compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -200,6 +201,7 @@ void buildFlowTransformPassPipeline(OpPassManager &passManager,
       // - Remove unit-extent dimensions.
       .addPass(mlir::createConvertElementwiseToLinalgPass)
       .addPass(mlir::createLinalgFoldUnitExtentDimsPass)
+      .addPass(createRaiseSpecialOps)
```

Comment:
Maybe better to move these also much earlier.... especially before const propagation.

Reply:
Running it after `createLinalgFoldUnitExtentDimsPass` allows catching more cases as it put the ops in a more canonical form. Otherwise some front ends create unit dims while some others don't